tools/distrib/pylint_code.sh

```diff
@@ -46,4 +46,10 @@ for dir in "${TEST_DIRS[@]}"; do
   $PYTHON -m pylint --rcfile=.pylintrc-tests -rn "$dir" || EXIT=1
 done
 
+find examples/python \
+  -iname "*.py" \
+  -not -name "*_pb2.py" \
+  -not -name "*_pb2_grpc.py" \
+  | xargs $PYTHON -m pylint --rcfile=.pylintrc-tests -rn
```

Comment:
Let's add a new pylint file for the examples directory rather than applying .pylintrc-tests here. There are a lot of exclusions for our tests that we probably don't expect users to make.